includes/blocks.inc

```diff
@@ -888,3 +896,124 @@ function islandora_solr_sort() {
 
   return $output;
 }
+
+/**
+ * Search results navigation block.
+ *
+ * Pull params out of the URL to determine the link to return
+ * the original search, and the previous and next object in the search
+ * according to the current object.
+ *
+ * @return string
+ *   The rendered html for the block content.
+ */
+function islandora_solr_search_navigation() {
```

Comment:
Talked with Nelson, he's going to add some inline docs to clarify how this function works.